[debug] [2022-08-29T16:37:56.667Z] ----------------------------------------------------------------------
[debug] [2022-08-29T16:37:56.669Z] Command:       C:\Program Files\nodejs\node.exe C:\Users\Market\AppData\Roaming\npm\node_modules\firebase-tools\lib\bin\firebase.js deploy
[debug] [2022-08-29T16:37:56.670Z] CLI Version:   11.8.0
[debug] [2022-08-29T16:37:56.670Z] Platform:      win32
[debug] [2022-08-29T16:37:56.670Z] Node Version:  v16.16.0
[debug] [2022-08-29T16:37:56.671Z] Time:          Mon Aug 29 2022 12:37:56 GMT-0400 (hora de Venezuela)
[debug] [2022-08-29T16:37:56.672Z] ----------------------------------------------------------------------
[debug] 
[debug] [2022-08-29T16:37:56.739Z] > command requires scopes: ["email","openid","https://www.googleapis.com/auth/cloudplatformprojects.readonly","https://www.googleapis.com/auth/firebase","https://www.googleapis.com/auth/cloud-platform"]
[debug] [2022-08-29T16:37:56.740Z] > authorizing via signed-in user (danietenos@gmail.com)
[debug] [2022-08-29T16:37:56.740Z] [iam] checking project cobuild-193ee for permissions ["cloudfunctions.functions.create","cloudfunctions.functions.delete","cloudfunctions.functions.get","cloudfunctions.functions.list","cloudfunctions.functions.update","cloudfunctions.operations.get","datastore.indexes.create","datastore.indexes.delete","datastore.indexes.list","datastore.indexes.update","firebase.projects.get"]
[debug] [2022-08-29T16:37:56.743Z] >>> [apiv2][query] POST https://cloudresourcemanager.googleapis.com/v1/projects/cobuild-193ee:testIamPermissions [none]
[debug] [2022-08-29T16:37:56.743Z] >>> [apiv2][(partial)header] POST https://cloudresourcemanager.googleapis.com/v1/projects/cobuild-193ee:testIamPermissions x-goog-quota-user=projects/cobuild-193ee
[debug] [2022-08-29T16:37:56.743Z] >>> [apiv2][body] POST https://cloudresourcemanager.googleapis.com/v1/projects/cobuild-193ee:testIamPermissions {"permissions":["cloudfunctions.functions.create","cloudfunctions.functions.delete","cloudfunctions.functions.get","cloudfunctions.functions.list","cloudfunctions.functions.update","cloudfunctions.operations.get","datastore.indexes.create","datastore.indexes.delete","datastore.indexes.list","datastore.indexes.update","firebase.projects.get"]}
[debug] [2022-08-29T16:37:58.011Z] <<< [apiv2][status] POST https://cloudresourcemanager.googleapis.com/v1/projects/cobuild-193ee:testIamPermissions 200
[debug] [2022-08-29T16:37:58.011Z] <<< [apiv2][body] POST https://cloudresourcemanager.googleapis.com/v1/projects/cobuild-193ee:testIamPermissions {"permissions":["cloudfunctions.functions.create","cloudfunctions.functions.delete","cloudfunctions.functions.get","cloudfunctions.functions.list","cloudfunctions.functions.update","cloudfunctions.operations.get","datastore.indexes.create","datastore.indexes.delete","datastore.indexes.list","datastore.indexes.update","firebase.projects.get"]}
[debug] [2022-08-29T16:37:58.012Z] >>> [apiv2][query] POST https://iam.googleapis.com/v1/projects/cobuild-193ee/serviceAccounts/cobuild-193ee@appspot.gserviceaccount.com:testIamPermissions [none]
[debug] [2022-08-29T16:37:58.012Z] >>> [apiv2][body] POST https://iam.googleapis.com/v1/projects/cobuild-193ee/serviceAccounts/cobuild-193ee@appspot.gserviceaccount.com:testIamPermissions {"permissions":["iam.serviceAccounts.actAs"]}
[debug] [2022-08-29T16:38:13.276Z] <<< [apiv2][status] POST https://iam.googleapis.com/v1/projects/cobuild-193ee/serviceAccounts/cobuild-193ee@appspot.gserviceaccount.com:testIamPermissions 200
[debug] [2022-08-29T16:38:13.276Z] <<< [apiv2][body] POST https://iam.googleapis.com/v1/projects/cobuild-193ee/serviceAccounts/cobuild-193ee@appspot.gserviceaccount.com:testIamPermissions {"permissions":["iam.serviceAccounts.actAs"]}
[info] 
[info] === Deploying to 'cobuild-193ee'...
[info] 
[info] i  deploying firestore, functions 
[info] Running command: npm --prefix "$RESOURCE_DIR" run lint
[debug] [2022-08-29T16:38:20.218Z] Error: Timed out.
    at Timeout._onTimeout (C:\Users\Market\AppData\Roaming\npm\node_modules\firebase-tools\lib\utils.js:253:49)
    at listOnTimeout (node:internal/timers:559:17)
    at processTimers (node:internal/timers:502:7)
[error] 
[error] Error: An unexpected error has occurred.
